src/Plugin/Block/ChatLeijuke.php

```diff
@@ -73,25 +73,28 @@ public function build() {
 
     foreach ($librariesYml as $k => $lib) {
       if ($k === $config['chat_selection']) {
```

Comment:
I'm not a fan of reading (or writing) code like this. In PHP (or any language for that matter) shortening the variables is not necessary as it usually will just confuse the one who tries to read the code later on. But I'll let this one pass for now. 😇

Reply:
I totally agree...